### ISSUE TITLE
target of inherits call cannot be in parentheses

### ISSUE THREAD

**Issue by fridis** [tokiwa-software/fuzion#3162]
This example declares two features `x` and `y` that both inherit from `(a u8).b`
```
a(A type) is
  b is

x :  a(u8).b is
y : (a u8).b is
```
but the declaration of `y` cannot be parsed
```
 > ./build/bin/fz test_it5.fz

/home/fridi/fuzion/work/test_it5.fz:5:14: error 1: Syntax error: expected semicolon ';', found keyword 'is'
y : (a u8).b is
-------------^
While parsing: semicolon or flat line break, parse stack: semiOrFlatLF, exprs, block, unit


/home/fridi/fuzion/work/test_it5.fz:5:1: error 2: Could not find called feature
y : (a u8).b is
^
Feature not found: 'y' (no arguments)
Target feature: 'universe'
In call: 'y'


/home/fridi/fuzion/work/test_it5.fz:4:1: error 3: Wrong number of type parameters
x :  a(u8).b is
^
Wrong number of actual type parameters in call:
Called feature: a.type
expected 2 generic arguments for 'THIS#TYPE, A'
found 1: 'a a.A'.

3 errors.

```

**Comment by simonvonhackewitz** [comment]
Even the first one does not work for me

```
a(A type) is
  b is

x : a(u8).b is
```

```
> FUZION_DISABLE_ANSI_ESCAPES=true fz /tmp/test.fz

/tmp/test.fz:4:1: error 1: Wrong number of type parameters
x : a(u8).b is
^
Wrong number of actual type parameters in call:
Called feature: a.type
expected 2 generic arguments for 'THIS#TYPE, A'
found 1: 'a a.A'.

one error.

```

**Comment by simonvonhackewitz** [comment]
duplicate of #1641, solved in #4393
